_posts/2019-08-04-bienvenue-ph-fr.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Bienvenue au Programming Historian en français!
+authors:
+- Sofia Papastamkou
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/images/welcome-ph-fr.png" /><figcaption>Schéma de Paul Otlet: documentation et télécommunication, <i>Mundaneum, Centre d'archives de la Fédération Wallonie-Bruxelles, Mons, Belgique. © Mundaneum</i></figcaption></figure></p>
```

Comment:
You have a broken image link - this should be `images/welcome-ph-fr/welcome-ph-fr.png`